CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ Here is a template for new release sections
 - Installation help for `pygraphviz` on Win10/64bit systems in `troubleshooting.rst` (#379)
 - LCOE_ASSET (Levelized Cost of Energy of Asset) explaination in KPI documentation (#458)
 - Heat demand profiles with option of using monitored weather data (ambient temperature) at the use case uvtgv. note: file not provided so far (#474)
+- Benchmark test for simple case grid and diesel without test for fuel consumption (#386)
```

Comment:
Hi @ursulaelmir ! I think you also changed some other stuff, didnt you? Fixing some of the warnings of the pytest? You should add that here as well.